### PR TITLE
PPC-Testing. Have crc32.S built as a lib with --tag=asm for builders that don't support autodiscover

### PR DESCRIPTION
**Pull request by daveh86**

### CODE REVIEW

**Comment by daveh86** [comment]
This request is just for testing at this stage.

Feel free to follow along at http://build.wiredtiger.com:8080/job/wiredtiger-test-format-stress-sanitizer-ppc/

**Comment by michaelcahill** [comment]
@daveh86, at minimum this needs a ticket and the title updated before it can be merged.

**Comment by daveh86** [comment]
@michaelcahill, Understood. This was just to allow the PPC testing with clang's sanitizer to compile. Is Clang compilation on PPC something we want to support longer term?

File: build_posix/Make.base [hunk 1]
```diff
@@ -6,10 +6,12 @@ ACLOCAL_AMFLAGS = -I build_posix/aclocal
 lib_LTLIBRARIES = libwiredtiger.la
 LDADD = $(lib_LTLIBRARIES)
 
+noinst_LTLIBRARIES = libpower8.la
```
Comment (agorrod) [inline build_posix/Make.base]: @daveh86 I'd rather this added a tag to the single file rather than creating a library.

**Comment by agorrod** [comment]
@daveh86 Can this be closed in favor of WT-2459?

**Pull request closed without merging**